content/intro-to-storybook/vue/fr/composite-component.md

```diff
@@ -23,125 +23,103 @@ Un composant composite n’est pas très différent des composants de base qu’
 
-```javascript
-//src/components/TaskList.stories.js
+```js:title=src/components/TaskList.stories.js
 import TaskList from './TaskList';
-import { taskData, actionsData } from './Task.stories';
+import * as TaskStories from './Task.stories';
 
-const paddedList = () => {
-  return {
-    template: '<div style="padding: 3rem;"><story/></div>',
-  };
-};
 export default {
+  component: TaskList,
   title: 'TaskList',
-  excludeStories: /.*Data$/,
-  decorators: [paddedList],
+  decorators: [() => '<div style="padding: 3rem;"><story /></div>'],
 };
 
-export const defaultTasksData = [
-  { ...taskData, id: '1', title: 'Task 1' },
-  { ...taskData, id: '2', title: 'Task 2' },
-  { ...taskData, id: '3', title: 'Task 3' },
-  { ...taskData, id: '4', title: 'Task 4' },
-  { ...taskData, id: '5', title: 'Task 5' },
-  { ...taskData, id: '6', title: 'Task 6' },
-];
-export const withPinnedTasksData = [
-  ...defaultTasksData.slice(0, 5),
-  { id: '6', title: 'Task 6 (pinned)', state: 'TASK_PINNED' },
-];
-
-// Etat de TaskList par défault
-export const Default = () => ({
+const Template = (args, { argTypes }) => ({
   components: { TaskList },
-  template: `<task-list :tasks="tasks" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
-  props: {
-    tasks: {
-      default: () => defaultTasksData,
-    },
-  },
-  methods: actionsData,
-});
-// Liste de tâches avec des tâches épinglées.
-export const WithPinnedTasks = () => ({
-  components: { TaskList },
-  template: `<task-list :tasks="tasks" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
-  props: {
-    tasks: {
-      default: () => withPinnedTasksData,
-    },
-  },
-  methods: actionsData,
-});
-// Liste des taĉhes en cours de chargement.
-export const Loading = () => ({
-  components: { TaskList },
-  template: `<task-list loading @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
-  methods: actionsData,
-});
-// Liste des taĉhes vide.
-export const Empty = () => ({
-  components: { TaskList },
-  template: `<task-list @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
-  methods: actionsData,
+  props: Object.keys(argTypes),
+  // Nous réutilisons nos actions de task.stories.js
+  methods: TaskStories.actionsData,
+  template: '<TaskList v-bind="$props" @pin-task="onPinTask" @archive-task="onArchiveTask" />',
 });
+
+export const Default = Template.bind({});
+Default.args = {
+  // Arranger les histoires à travers la composition d'arguments.
+  // Les données ont été héritées de l'histoire Default dans task.stories.js.
+  tasks: [
+    { ...TaskStories.Default.args.task, id: '1', title: 'Task 1' },
+    { ...TaskStories.Default.args.task, id: '2', title: 'Task 2' },
+    { ...TaskStories.Default.args.task, id: '3', title: 'Task 3' },
+    { ...TaskStories.Default.args.task, id: '4', title: 'Task 4' },
+    { ...TaskStories.Default.args.task, id: '5', title: 'Task 5' },
+    { ...TaskStories.Default.args.task, id: '6', title: 'Task 6' },
+  ],
+};
```

Comment:
@jeanphibaconnais if you don't mind leaving the comments in the snippets in their native form (English) I'd appreciate it! I know that it brings the translation "closer to home" (pardon the bad pun), but until I figure out a better way for this it would be best to leave them at that!

Reply:
No problem ! Do you want i change all the pages now ?

Reply:
i removed the yarn.lock file, changed the title and the comment on the snippet in composite-component page.

I'd tryed to translate comments and test phrase but do you prefer that all snippet code is in english ?

Reply:
@jeanphibaconnais yes all the snippets if you don't mind! Sorry if I wasn't clear enough!

Reply:
hi @jonniebigodes , i've just changed that (and some changes in diff blocks).

Reply:
hi @jonniebigodes it's ok for you this conversation ?